src/settings/index.js

```diff
@@ -137,6 +138,14 @@ const pages = [
     component: OkapiTimers,
     perm: 'ui-developer.settings.okapiTimers',
   },
+  {
+    route: 'rtr',
+    labelId: 'ui-developer.rtr',
+    component: RefreshTokenRotation,
+    // FIXME use permissions for rtr
+    // perm: 'ui-developer.settings.rtr',
```

Comment:
Does this still need to be commented out?

Reply:
Oh, good catch, @BogdanDenis. It should stay commented out while people are playing with this locally since that permission won't be installed to Okapi until the PR is merged, but it definitely needs to be un-commented then!